modules/rosa-list-objects.adoc

```diff
@@ -77,6 +77,41 @@ $ rosa list clusters [arguments]
 |Specifies an AWS profile (string) from your credentials file.
 |===
 
+[id="rosa-list-break-glass-credential_{context}"]
+== list break glass credentials
+
+Lists all of the break glass credentials for a cluster.
+
+.Syntax
+[source,terminal]
+----
+$ rosa list break-glass-credential  [arguments]
+----
+
+.Arguments
+[cols="30,70"]
+|===
+|Option |Definition
+
+|--cluster <cluster_name>\|<cluster_id>
+|Required. The name or ID of the cluster to which the break glass credentials have been added.
+|===
+
+.Optional arguments inherited from parent commands
+[cols="30,70"]
+|===
+|Option |Definition
+
+|--help
+|Shows help for this command.
+
+|--debug
+|Enables debug mode.
+
+|--profile
+|Specifies an AWS profile (string) from your credentials file.
+|===
+
```

Comment:
Here you can add an example command to list all break-glass-credential for a cluster named mycluster
$ rosa list break-glass-credential --cluster=mycluster

Also you didn't add the describe command for break-glass-credential